omero/sysadmins/unix/server-debian9-ice36.rst

```diff
@@ -54,6 +54,12 @@ To install Ice 3.6:
     :start-after: #start-recommended-ice
     :end-before: #end-recommended-ice
 
+As part of the installation of Ice, two new daemons are installed and automatically started.
+They both can be safely disabled by running the following command::
```

Comment:
They *should* be if you're wanting `bin/omero admin start` to do anything useful.

Reply:
Are you suggesting to rephrase?

Reply:
Yes: as it is, it looks optional, a matter of user preference.